omod/src/main/webapp/pages/conditionlist/addCondition.gsp

```diff
@@ -58,8 +58,8 @@
                         formFieldName: "conditionStartDate",
                         label        : "",
                         useTime      : false,
-                        defaultDate: new Date(),
-                        endDate    : new Date(),
+                        defaultDate  : new Date(),
+                        startDate    : new Date()
```

Comment:
Establish that because in my opinion, we shouldn't save condition in future date